Algorithm/Bit_manipulation/readme.md

```diff
@@ -52,4 +52,7 @@ There are different bitwise operations used in the bit manipulation. Bit operati
 
 
 ## Questions :
-- Brian Kernighan's Algorithm ----> [C++](/Code/C++/Brian_Kernighan.cpp) 
\ No newline at end of file
+
+* Brian Kernighan's algorithm --> [Java](/Code/Java/BrianKernighanAlgorithm.java)
+- Brian Kernighan's Algorithm ----> [C++](/Code/C++/Brian_Kernighan.cpp) 
```

Comment:
heyy, this has to be written like
```suggestion
- Brian Kernighan's Algorithm ----> [C++](/Code/C++/Brian_Kernighan.cpp) | [Java](/Code/Java/BrianKernighanAlgorithm.java)
```
and then delete line 56

Reply:
Done